pymc3/distributions/continuous.py

```diff
@@ -1134,27 +1134,13 @@ def logcdf(self, value):
         a = normal_lcdf(0, 1, (q - 1.0) / r)
         b = 2.0 / l + normal_lcdf(0, 1, -(q + 1.0) / r)
 
-        left_limit = (
-            tt.lt(value, 0)
-            | (tt.eq(value, 0) & tt.gt(mu, 0) & tt.lt(lam, np.inf))
-            | (tt.lt(value, mu) & tt.eq(lam, 0))
-        )
-        right_limit = (
-            tt.eq(value, np.inf)
-            | (tt.eq(lam, 0) & tt.gt(value, mu))
-            | (tt.gt(value, 0) & tt.eq(lam, np.inf))
-        )
-        degenerate_dist = (tt.lt(mu, np.inf) & tt.eq(mu, value) & tt.eq(lam, 0)) | (
-            tt.eq(value, 0) & tt.eq(lam, np.inf)
-        )
-
         return bound(
             tt.switch(
-                ~(right_limit | degenerate_dist),
-                a + tt.log1p(tt.exp(b - a)),
+                tt.lt(value, np.inf),
```

Comment:
For example `pm.Wald.dist(mu=1, lam=1).logcdf(1e310)`, will return 0 instead of `nan` this way. This behavior seems in line with other distributions I tested.